models/pardot/base.sql

```diff
@@ -1,8 +1,8 @@
-drop schema if exists pardot cascade;
-create schema pardot;
+drop schema if exists pardot_analysis cascade;
```

Comment:
I've been thinking about that those create schema commands. I don't think this is a good place for them, if they are in fact needed. I'm going to remove them and we can figure out about construction / teardown of these views later.